xpallets/system/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+use sp_runtime::traits::StaticLookup;
```

Comment:
I prefer to place the part of the standard library separately at the top of the imports